### DISCUSSION TITLE
Hello world: new/prospective members' introduction/welcome thread

### DISCUSSION THREAD

**Discussion by sanjayankur31** [OCNS/SoftwareWG#12] [General]
This thread is for new/prospective members to introduce themselves to the community, and for us to welcome them.

Please share this with anyone who may be interested in participating in the community.

**Comment by sanjayankur31** [comment]
cc: @OCNS/software-wg 

Hi folks,

Please welcome @dhasegan to the community. @dhasegan , please introduce yourself so that everyone can get to know you when you have a minute, and of course, please feel free to start new discussions etc.

**Reply by dhasegan** [reply]
Hi everyone, I've actually never used Discussions on Github, pretty cool. 
I am @dhasegan , I'm an industry engineer in Machine Learning for Healthcare. I've built ML infrastructure at Twitter and at Braid Health(currently ~ 10 people startup), and I want to learn and contribute to Computational Neuroscience Tooling. 
Are there any some low hanging fruits that I can help with right away? Like install travis on a package, stuff like that?

**Reply by sanjayankur31** [reply]
I think most projects that the folks of the WG here work on probably have CI pipelines set up already, either on Travis or GitHub actions.

We don't have a central tracker for tasks at the moment.  I've been thinking of whether we need to have a centralised list of tasks where developers can add tasks they are looking for help with, and folks looking to get involved could use. Something like https://fedoraproject.org/easyfix/ ? It could be a repo where folks manually add tasks and link to project issues, or we could also automate it so that issues with a particular tag (help-wanted/easyfix/need-help/good-first-issue) in added projects are automatically listed there (like the above link).

Thoughts on this? Worth discussing in a different discussion/ticket?

**Reply by dhasegan** [reply]
I think that having a centralized list of tasks would be useful for keeping track of bugs, small fixes, or TODOs. Then people can see that and coordinate independently. You can see such workflows in big open source projects like React ( https://github.com/facebook/react/issues ).

**Reply by sanjayankur31** [reply]
Each repository does keep a list of issues---take a look at the repos the working group members maintain? The tasks that the working group undertakes are in the issues on this repo.

I was wondering if we could consolidate simpler issues from various neuroscience into a single page/issue :thinking:

**Reply by joewgraham** [reply]
> Hi everyone, I've actually never used Discussions on Github, pretty cool.
> I am @dhasegan , I'm an industry engineer in Machine Learning for Healthcare. I've built ML infrastructure at Twitter and at Braid Health(currently ~ 10 people startup), and I want to learn and contribute to Computational Neuroscience Tooling.
> Are there any some low hanging fruits that I can help with right away? Like install travis on a package, stuff like that?

Hi Daniel, I'm a neuroscientist and developer of NetPyNE (www.netpyne.org, https://github.com/Neurosim-lab/netpyne), an open-source Python package to facilitate the development, parallel simulation, analysis, and optimization of biophysical neuronal networks using the NEURON simulator.

There's always work to be done around here and we'd be glad for the help!  We are planning on switching from Travis CI to GitHub Actions for our testing.  If you'd be interested in working on that (or any other low-hanging fruit), please let me know.

**Reply by dhasegan** [reply]
Hi @joewgraham , I am definitely going to explore NetPyNE to see where I can help. I also chatted with Salvador at INCF and I joined the slack channel for NetPyNE contributors. Tty there. Thanks!

**Comment by salvadord** [comment]
Hello! My name is Salvador Dura-Bernal (lab web: http://dura-bernal.org) and I would be happy to give a talk presenting the NetPyNE tool (www.netpyne.org). thanks

**Reply by sanjayankur31** [reply]
Welcome to the community Salva :)

We have a ticket open for a NetPyNE dev session here: https://github.com/OCNS/SoftwareWG/issues/9

(can't tag you in it because you're not part of the SoftwareWG GitHub team yet---should I add you to it?)

**Reply by salvadord** [reply]
thanks ankur! I see joe is already on it, so that's fine, he can coordinate this, thanks

**Reply by sanjayankur31** [reply]
Sounds good. If you folks have any dates in mind, please do let us know.

**Comment by akashmishra23** [comment]
Hi Everyone, I'm Akash, 

Currently a Deep Learning Engineer and Solutions Architect in healthcare, primarily working with Medical Imaging but mostly focused on Neuroimaging Analysis on multiple modalities

I've been fascinated by Computational Neuroscience and Neuroinformatics for a long time now and want to contribute to the effort any way i can

**Reply by sanjayankur31** [reply]
Welcome again, @akashmishra23

**Reply by tnowotny** [reply]
Welcome, Akash!
I don't know whether you are looking to do some hacking but we have some interesting open issues if you want to get stuck into that kind of thing.
https://github.com/genn-team/genn/issues

**Reply by akashmishra23** [reply]
@tnowotny  I'm primarily proficient in DevOps, Deep Learning on PyTorch, Tensorflow and flux, as well as Standardization of libraries, best practises, etc given that i also hold experience in Neuroimaging Analysis using nipy, nilearn, etc, I'm planning on going through the projects and issues over the weekend,

GeNN looks really interesting, I'll go through the issues today

**Comment by juhuntenburg** [comment]
Hi! 
I am Julia, DevOps engineer with the International Brain Laboratory (https://www.internationalbrainlab.com/). Before joining IBL I did my own research and tool development. mostly working with high-res MRI data. I am here to make sure IBL is connected to what is going on in the community regarding software development. And if I have some space of course happy to help out :)

**Reply by sanjayankur31** [reply]
Welcome again @juhuntenburg , great to have you here. :sparkle:

**Reply by sanjayankur31** [reply]
Quick follow up: would you be up for a dev session on DevOps in general, or how it's implemented at the IBL at some point? These are very informal---one doesn't even need to make slides. You can see recordings from past dev sessions here: https://ocns.github.io/SoftwareWG/category/events/

**Comment by Dextromethorpan** [comment]
Hi everybody, my name is @Dextromethorpan. I am Luciano Muratore. I am a telecommunication engineer who just moved to Brussels.
I am interested in Neuroinformatics and its applications to Memory.
I am new in this industry, so I am willing to learn and to participate.
Thanks.

**Reply by sanjayankur31** [reply]
Welcome to the community @Dextromethorpan . I've invited you to the GitHub team now. 

Most members are on leave/holidays for the next month or two, so we'll restart our activities towards the end of September

https://ocns.github.io/SoftwareWG/2021/07/19/summer-vacations.html

But quite a few of us will remain active asynchronously here over GitHub, so if there's anything at all you'd like to discuss, please do open issues/discussion topics.

**Reply by Dextromethorpan** [reply]
Hi there, 
thanks, I will keep that in mind.

**Comment by Athene-ai** [comment]
Hi all!

I am Daniela, an Italian computational neuroscientist.

I can not wait to working with you all!

**Reply by sanjayankur31** [reply]
Hi @Athene-ai , welcome to the community! We're all looking forward to working with you too :)

**Reply by Athene-ai** [reply]
> Hi @Athene-ai , welcome to the community! We're all looking forward to working with you too :)

The pleasure is mine!

**Comment by imeMFK01** [comment]
Hello everyone,

I am Muhammad Farhan Khalid ([@iFarhan](https://github.com/imeMFK01)), a research assistant at Lahore University of Management Sciences, with a passion for tool development.

I am delighted to be part of this community. Please do not hesitate to reach out if there is anything I can assist you with or if you would like to discuss anything further.

Looking forward to engaging with you all.

Thank you.

**Reply by sanjayankur31** [reply]
Welcome @iFarhan, great to have you here. Do have a look at the issues on this repo---that'll give you an idea of what's going on. We're going to restart our meetings in May, so we hope to see you there. Please feel free to use the discussions etc. to share information with the community/start new discussion topics and so on.

**Reply by imeMFK01** [reply]
Sure will definitely look into the issues and excited to join our upcoming meetings.
